modules/lambda/variables.tf

```diff
@@ -25,3 +25,9 @@ variable "alarm_topic_arn" {
   type        = string
   description = "ARN of SNS Topic, for alarm notifications"
 }
+
+variable "dlq_enabled" {
+  type        = bool
+  description = "dead letter config enabled"
```

Comment:
This is the only thing I can see here I would like to see improved--this much I can discern from the name of the variable. More description about what it is used for would help.